scripts/std-lib-regression.sh

```diff
@@ -68,7 +67,7 @@ echo "Starting cargo build with Kani"
 export RUST_BACKTRACE=1
 export RUSTC_LOG=error
 export KANIFLAGS="--goto-c --ignore-global-asm --reachability=legacy"
-export RUSTFLAGS="--kani-flags"
+export RUSTFLAGS="--kani-flags -C panic=abort"
```

Comment:
Why this change?

Reply:
I had to change this because in this script we are invoking `kani-compiler` via `cargo build` not via `cargo kani`. This is the only relevant flag for the codegen to go through.